src/js/player.js

```diff
@@ -68,8 +68,9 @@ function playVideo(videoId, playlistId = '') {
         playerView.channelName = data.author;
         playerView.channelId = data.authorId;
         playerView.channelIcon = data.authorThumbnails[2].url;
-
+        
         let videoUrls = data.formatStreams;
+        let formatUrls = data.adapativeFormats;
```

Comment:
This looks like it's spelled incorrectly.  Please fix this and everything should be good to go. :)